src/app/ApplicationTemplate.UWP/Views/Styles/Application/Converters.xaml

```diff
@@ -27,4 +27,20 @@
 
 	<converters:DebugConverter x:Key="debug" />
 
+	<!-- Create  -->
+	<converters:FromPasswordStateToCustomValueConverter x:Key="PasswordStateToIconStyle"
```

Comment:
Those converters seem very specific to the CreateAccountPage. I think they should be located over there rather than global to the whole app.